changelog/changelog.ddoc

```diff
@@ -15,22 +15,36 @@ SUBNAV=$(SUBNAV_TEMPLATE
 SEARCH_OPTIONS_EXTRA=<option value="dlang.org/changelog" selected>Change
 Log</option>
 
-VERSION=
+STABLE_VERSION=
 $(DIVC version,
 $(P
 $(B $(LARGE $(LINK2 http://downloads.dlang.org/releases/2.x/$(VER), Download D $(VER))))$(BR)
 $(SMALL released $1, $2)
 )
 $4
 )
+VERSION=$(STABLE_VERSION $0)
+
+BETA_VERSION=
+$(DIVC version,
+$(P
+$(B $(LARGE $(LINK2 http://downloads.dlang.org/releases/2.x/$(VER), Download D$(VER) (beta))))$(BR)
+$(SMALL to be released $1, $2)
+$(P Please report bugs at $(LINK2 $(ROOT_DIR)bugstats.html, Bugzilla).)
+$(MESSAGE_BOX orange, Experimental - beta releases may contain regressions. Test with care.)
+)
+$4
+)
 
 NIGHTLY_VERSION=
 $(DIVC version,
 $(P
-$(B $(LARGE $(LINK2 http://nightlies.dlang.org, Download D nightlies)))$(BR)
+$(B $(LARGE $(LINK2 http://nightlies.dlang.org, Download D (Nightly))))$(BR)
 $(SMALL $1)
+$(P Please report bugs at $(LINK2 $(ROOT_DIR)bugstats.html, Bugzilla).)
+$(MESSAGE_BOX red, Unstable - nightly releases are not suited for production. Use stable.)
 )
-$4
+$3
```

Comment:
In https://github.com/dlang/tools/pull/251 the format is unified, that's why the tools PR needs to be pulled first.